web/src/lib/components/admin-page/settings/admin-settings.svelte

```diff
@@ -35,6 +36,8 @@
       savedConfig = cloneDeep(newConfig);
       notificationController.show({ message: 'Settings saved', type: NotificationType.Info });
 
+      await loadConfig();
```

Comment:
Somewhat outside the scope of this PR, but added this to ensure the system settings are reloaded after they are saved. This is used in the confirmation screen when deleting a user. Without this change, updating the setting and then removing a user without reloading the page will still show the old value.

Reply:
newConfig should be the latest config. Maybe we just need to update the store with the new value.

Reply:
Any examples of that being done somewhere? I tried replacing this with 
```
serverConfig.update(() => ({ ...savedConfig, loaded: true }));
``` 
Also tried newConfig but the delete confirm modal shows undefined after making a settings update and not refreshing page.

Reply:
Ah, its different, one is `SystemConfigDto` and the other is `ServerConfigDto`

Reply:
Ok, this works for my specific field and saves the network call. But does not refresh any other fields. What do you think, network call and ensure everything is up to date, or pick and choose what is updated?

```
serverConfig.update(() => ({ userDeleteDelay: newConfig.user.deleteDelay, loaded: true }));
```

Reply:
Oh right, this is just the public config. You can leave the load config in for now. I think the server can emit an event through the websocket to push out the update though, but it doesn't really matter.